src/components/CommentCard.jsx

```diff
@@ -0,0 +1,20 @@
+import { dateFormatter } from "../utils/formatters";
+
```

Comment:
It was impossible to navigate to this page when I opened it without going to your app file, looking for the correct path and then entering it manually. Can you have a think about the user's journey through the site (starting at `/`) and how they can get to an article? Loading patterns would also help the user understand if they're in the right place or not.